overlay_all_stats.py

```diff
@@ -32,86 +32,112 @@ def _update_data_layer(self):
         self.data_canvas.clear()
         self.message_canvas.clear()
 
-        if self.data["power"] != 0:
+        if self.data["power"].get() != 0:
```

Comment:
So the reason we have all these checks to see if values are 0 is because 0 used to be the default value for data. Now that we instead can check if data is valid (which the code now does later on), we don't need to do these checks! So, you can remove all the `if`s and call the functions directly - they'll only draw if the data is valid.